### PR TITLE
Update SendToKindle

### PR DESCRIPTION
**Pull request by Ephemera**

### CODE REVIEW

**Comment by vitorgalvao** [comment]
We prefer versioned casks. Why are you changing it, are versioned downloads no longer provided?

File: Casks/send-to-kindle.rb [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
   # amazonaws.com is the official download host per the vendor homepage
-  url "https://s3.amazonaws.com/sendtokindle/SendToKindleForMac-installer-v#{version}.pkg"
+  url "https://s3.amazonaws.com/sendtokindle/SendToKindleForMac-installer.pkg"
   name 'Send to Kindle'
   homepage 'https://www.amazon.com/gp/sendtokindle/mac'
   license :gratis
 
-  pkg "SendToKindleForMac-installer-v#{version}.pkg"
+  pkg "SendToKindleForMac-installer.pkg"
```
Comment (vitorgalvao) [inline Casks/send-to-kindle.rb]: Double quotes are reserved for cases where string interpolation is needed. Please change these to single quotes.

**Comment by Ephemera** [comment]
@vitorgalvao 
That's right. They are no longer provide.

**Comment by vitorgalvao** [comment]
Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by Ephemera** [comment]
@vitorgalvao 
Done :]

**Comment by vitorgalvao** [comment]
Thank you for the contribution. It was merged directly as 7385208b948c7c6a43ae585323ddeecd2565cbe0 to keep commit history cleaner. Your contribution is still credited to you.

**Pull request closed without merging**